mapwidgets/templates/mapwidgets/google-point-field-widget.html

```diff
@@ -22,7 +22,7 @@
                 <a class="mw-btn mw-btn-info mw-btn-my-location" >{% trans "Use My Location" %}</a>
                 <a class="mw-btn mw-btn-warning mw-btn-add-marker" ><i class="icon-location"></i> <span class="point-text">{% trans "Choose point on map" %}</span></a>
                 <div class="mw-adress-input-wrap pull-right">
-                    <input type="text" class="form-control pull-right" id="{{ name }}-mw-google-address-input" role="presentation" autocomplete="off" placeholder="{% trans 'Find a Location by Address' %}" />
+                    <input type="text" class="form-control pull-right" id="{{ name }}-mw-google-address-input" name="{{ name }}-mw-google-address-input" role="presentation" autocomplete="off" placeholder="{% trans 'Find a Location by Address' %}" />
```

Comment:
Thank you for your contribution. 

This input shouldn't be posted, it is just for autocompletion usage. Please revert this line change, I am OK with the rest of your PR changes.